extensions/wikia/Recirculation/js/scrolldepth.js

```diff
@@ -0,0 +1,35 @@
+/*global require*/
+require([
+	'jquery',
+	'wikia.tracker',
+	'wikia.abTest',
+	'ext.wikia.recirculation.views.incontent'
+], function ($, tracker, abTest, incontent) {
+	'use strict';
+
+	if (!abTest.inGroup('RECIRCULATION_SCROLLDEPTH', 'YES')) { return; }
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reformat the code to have only one statement per line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AOneStatementPerLine)